lib/trento_web/views/host_view.ex

```diff
@@ -0,0 +1,28 @@
+defmodule TrentoWeb.HostView do
+  use TrentoWeb, :view
+
+  def render("hosts.json", %{hosts: hosts}) do
+    render_many(hosts, __MODULE__, "host.json")
+  end
+
+  def render("host.json", %{host: host}), do: host
```

Comment:
Absolutely, you are right, this was made before the other views, before I started converting to a map directly, that view works because there are all loaded association where it's used, so the jason encoder of the schema works without hassles, but I will make that explicit converting to a map and inherit that. This works for a side effect. Thanks for the catch!

Reply:
done